scripts/lmdk/tools/cmake_build.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: BSD-3-Clause
+
+
+import pathlib
+import dataclasses
+from .utils import rmtree_if_exists, execute_command
+
+
+SOF_TOP = pathlib.Path(__file__).parents[3].resolve()
```

Comment:
This looks like overwriting a global from the caller... is it not? If not then maybe call it _SOF_TOP to make the difference clearer.